src/Transition.js

```diff
@@ -590,3 +575,17 @@ Transition.ENTERED = ENTERED
 Transition.EXITING = EXITING
 
 export default Transition
+
+function getTimeouts(timeout) {
```

Comment:
**suggestion:** it would probably be a bit cleaner to refactor this to also take the _type_ of timeout desired – e.g. there's no need to calculate the "exit" timeout in the "enter" flow, and no need to calculate the "enter/appear" timeout in the "exit" flow

just would be nice to clean up APIs as we're refactoring.

Reply:
Adding the type argument I think will add more logic to the function, I splitted better in 2 functions.